server/matchmaker/map_pool.py

```diff
@@ -50,10 +52,13 @@ def choose_map(self, played_map_ids: Iterable[int] = ()) -> Map:
                 least_common = least_common[:i]
                 break
 
-        weights = [self.maps[id_].weight for id_, _ in least_common]
+        least_common_ids = {id_ for id_, _ in least_common}
 
-        map_id = random.choices(least_common, weights=weights, k=1)[0][0]
-        return self.maps[map_id].get_map()
+        # Multiply weight by 2 if map is least common and not vetoed by anyone
+        mapList = list((map.map_pool_map_version_id, map, 2 if (map.id in least_common_ids) and (vetoesMap.get(map.map_pool_map_version_id, 0) == 0) else 1) for id, map in self.maps.items())
```

Comment:
don't you feel like this line is a bit long?)

Reply:
long lines for long PP